src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -146,6 +148,7 @@ const usePushNotificationHook = () => {
     }
     if (notification.data.type === 'message.new') {
       if (notification.userInteraction) {
+        await refetchMessage(); // refetch message
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refetchMessage()` here seems to be intended to update the chat messages when a new message notification is interacted with by the user. However, it's important to consider error handling for this function call. If the refetch fails due to network issues or server errors, the user might end up seeing outdated data. Consider adding error handling or retry logic to improve the robustness of this feature.
```diff
+ await refetchMessage(); // refetch message
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->